### ISSUE TITLE
Dynamic location appearance of "Nipple"

### ISSUE THREAD

**Issue by shaylevary** [caiiiycuk/js-dos.com#241]
Hi

I've noticed that in some of the mobile games there is an option of dynamic location appearance of the nipple (virtual gamepad).

I've tried to find this feature on the dos zone studio, but there is no dynamic nipple activator feature.

This feature appears on this game for example:
https://dos.zone/prince-of-persia-1990/

Is there any option to add the dynamic nipple feature to the dos zone studio?

Thanks in advance.
![Screenshot_20220806-162830_Chrome](https://user-images.githubusercontent.com/10597517/183251246-83da2ce0-e6cb-4895-a307-2632169c7286.jpg)

**Comment by caiiiycuk** [comment]
Yeah, that interesting. But if you have two nipples which one should be activated?

**Comment by shaylevary** [comment]
Hi,

You are right, I've tried to rechecked this issue and it does create some problems, for example if I press the action keys in the virtual gamepad, the nipple appears there and blocks the presses.
Anyway, Perhaps it is possible to define that it can be activated only in the case of one nipple and as long as its workspace remains in the left/right area of ​​the screen (on the opposite side to the location of the gamepad buttons)
